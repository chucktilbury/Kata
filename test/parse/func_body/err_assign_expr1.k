;-------------------------------------------------------------------------------
; Assignemnet is missing the expression
;-------------------------------------------------------------------------------

var string name1 = 

