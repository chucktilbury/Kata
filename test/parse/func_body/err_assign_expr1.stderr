Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_assign_expr1.k: 7: 1: expected an assignment item but got end of file
